dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/publish/servers/TxFaultToleranceReordered/server.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+    Copyright (c) 2018 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server>
+
+    <include location="../fatTestPorts.xml"/>
+    
+	<featureManager>
+	    <feature>componenttest-1.0</feature>
+		<feature>osgiconsole-1.0</feature>
+		<feature>localConnector-1.0</feature>
+		<feature>appSecurity-3.0</feature>
+		<feature>cdi-2.0</feature>
+		<feature>servlet-4.0</feature>
+		<feature>mpFaultTolerance-1.0</feature>
+	</featureManager>
+
+    <logging traceSpecification="Transaction=all=enabled:FAULTTOLERANCE=all=enabled:logservice=all=enabled:*=info=enabled:com.ibm.websphere.microprofile.faulttolerance.*=all=enabled:com.ibm.ws.threading.Policy*=all:com.ibm.ws.threading.internal.Policy*=all:com.ibm.ws.threading.internal.ThreadPoolController=event"/>
```

Comment:
Is all this trace really needed? too much trace was partly to blame for the cloud issues we've seen over the last few weeks. There has been a big push to turn it off unless there is a known defect that needs diagnosis.

Reply:
Trace reduced.